guides/doc-Administering_Red_Hat_Satellite/topics/Renaming_a_Capsule_Server.adoc

```diff
@@ -4,28 +4,35 @@
 The host name of {SmartProxyServer} is referenced by {ProjectServer} components, and all hosts registered to it.
 This procedure ensures that you update all references to the new host name.
 
+[IMPORTANT]
+====
+* Both the `hostname` and `hostname -f` commands must return the FQDN of {SmartProxyServer} or the `{project-change-hostname}` script will fail to complete.
+
+* If the `hostname` command returns the shortname of {SmartProxyServer} instead of the FQDN, use `hostnamectl set-hostname _<old_fqdn>_` to set the old FQDN correctly before attempting to use the `{project-change-hostname}` script.
+====
+
 .Prerequisites
+
+* Backup {SmartProxyServer}.
+The `{project-change-hostname}` script makes irreversible changes to {SmartProxyServer}.
+If the renaming process is not successful, you must restore it from a backup.
 +
+{ProjectName} does not provide a native backup method for {SmartProxyServer}.
+For more information, see xref:backing-up-satellite-server-and-capsule-server[].
+
 [WARNING]
 ====
 Until https://bugzilla.redhat.com/show_bug.cgi?id=1829115[BZ#1829115] is resolved, you must edit the `usr/share/katello/hostname-change.rb` file on {SmartProxyServer} and comment out the following lines before attempting to rename {SmartProxyServer}:
 
-[options="nowrap" subs="+quotes,attributes"]
+[options='nowrap', subs="+quotes,verbatim,attributes"]
 ----
 STDOUT.puts "updating hostname in hammer configuration"
-self.run_cmd("sed -i.bak -e 's/#{@old_hostname} \
-/#{@new_hostname}/g' #{hammer_root_config_path}/*.yml")
-self.run_cmd("sed -i.bak -e 's/#{@old_hostname} \
-/#{@new_hostname}/g' #{hammer_config_path}/#.yml")
+self.run_cmd("sed -i.bak -e 's/\#_@<old_hostname>_ \
+/_#@<new_hostname>_/g' _\#<hammer_root_config_path>_/\*.yml")
+self.run_cmd("sed -i.bak -e 's/#_@<old_hostname>_ \
+/#_@<new_hostname>_/g' #_<hammer_config_path>_/*.yml")
```

Comment:
Can you explain this change?

Reply:
I am also not following, was this not working at all?

Reply:
Please remove the brackets as we use italics for variables throughout our docs, so the underscores should suffice. Thanks!